docs/src/create_reference_md.py

```diff
@@ -168,76 +168,78 @@ def metric_source_slug(metric_name: str, source_name: str) -> str:
     return slugify(f"{metric_name} from {source_name}")
 
 
-def metric_source_section(data_model, metric_key: str, source_key: str) -> str:
+def metric_source_section(metric_key: str, source_key: str) -> str:
     """Return the metric source combination as Markdown section."""
-    source = data_model["sources"][source_key]
-    source_name = source["name"]
-    metric = data_model["metrics"][metric_key]
-    metric_name = metric["name"]
-    metric_link = f"[{metric_name.lower()}]({slugify(metric_name)})"
-    source_link = f"[{source_name}]({slugify(source_name)})"
+    source = DATA_MODEL.sources[source_key]
+    metric = DATA_MODEL.metrics[metric_key]
+    metric_link = f"[{metric.name.lower()}]({slugify(metric.name)})"
+    source_link = f"[{source.name}]({slugify(source.name)})"
     markdown = markdown_paragraph(f"{source_link} can be used to measure {metric_link.lower()}.")
-    if documentation := source.get("documentation", {}).get(metric_key):
+    if documentation := source.documentation and source.documentation.get(metric_key):
```

Comment:
Does the walrus operator work on a single segment? I would probably put parenthesis around the first part to be sure

Reply:
I'll put in the parentheses, unless we get some linter complaining about unnecessary parentheses :-)